Keybinds.cpp

```diff
@@ -186,6 +186,7 @@ action_name_mapper actionnamemap[] = {
     {"TOGGLE_STOCKS", action_togglestockpiles},
     {"TOGGLE_ZONES", action_togglezones},
     {"TOGGLE_OCCLUSION", action_toggleocclusion},
+    {"TOGGLE_FOG",action_togglefog},
```

Comment:
what keys are these new features bound to? I'd make those modifications to the keybinds file in this PR since we may not get to #135 before the release this Friday.